app/views/contexts/summary/question.py

```diff
@@ -95,10 +108,58 @@ def _build_answers(
             ]
 
         summary_answers = []
+
+        if self.dynamic_answer_schemas:
+            for answer_schema in self.dynamic_answer_schemas:
+                for list_item_id in self.list_store.get(
+                    self.dynamic_answer_identifier
+                ).items:
+                    placeholder_renderer = PlaceholderRenderer(
+                        answer_store=self.answer_store,
+                        list_store=self.list_store,
+                        progress_store=self.progress_store,
+                        schema=self.schema,
+                        language="en",
+                        metadata=None,
+                        response_metadata={},
```

Comment:
These need to be passed in, not empty/none.

Reply:
Do we need metadata/response metadata for dynamic answers placeholders, I thought we only need list/answer sources for labels here?